tabular/src/autogluon/tabular/features/generators/auto_ml_pipeline.py

```diff
@@ -62,8 +65,10 @@ class AutoMLPipelineFeatureGenerator(PipelineFeatureGenerator):
     >>>
     >>> X_test_transformed = feature_generator.transform(test_data)
     """
-    def __init__(self, enable_numeric_features=True, enable_categorical_features=True, enable_datetime_features=True,
-                 enable_text_special_features=True, enable_text_ngram_features=True, vectorizer=None, **kwargs):
+    def __init__(self, enable_numeric_features=True, enable_categorical_features=True,
+                 enable_datetime_features=True,
+                 enable_text_special_features=True, enable_text_ngram_features=True,
+                 enable_raw_text_features=True, vectorizer=None, **kwargs):
```

Comment:
`enable_raw_text_features=False` by default, to not impact normal users while we work on adding multimodal support. This will also fix CI build.